app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -625,6 +628,16 @@ private double getPokemonLevel(Bitmap pokemonImage){
         }
         return 1;
     }
+
+    /**
+     * Get the hashcode for a bitmap
+     */
+    String hashBitmap(Bitmap bmp) {
+        int[] allpixels = new int[bmp.getHeight() * bmp.getWidth()];
+        bmp.getPixels(allpixels, 0, bmp.getWidth(), 0, 0, bmp.getWidth(), bmp.getHeight());
+        return Integer.toHexString(Arrays.hashCode(allpixels));
```

Comment:
Why `toHexString`? That seems just a way to waste memory. `LruCache<int, String>` might not work, but `LruCache<Integer, String>` will.